tests/e2e/smoke-statefulset/00-assert.yaml

```diff
@@ -16,3 +16,24 @@ status:
   scale:
     replicas: 1
     statusReplicas: "1/1"
+---
```

Comment:
We should also assert that `serviceName` is set to `stateful-collector-headless` in the StatefulSet definition above.